feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/README.md

```diff
@@ -4,33 +4,32 @@
 
 BGP Graceful Restart
 
-## Procedure
+## Topology
+Follwing connections:
+ *   ATE port-1 <--> DUT port-1 
+ *   ATE port-2 <--> DUT port-2
 
-*   Establish eBGP sessions between:
-    *   ATE port-1 and DUT port-1
-    *   ATE port-2 and DUT port-2
-    *   Configure allow route-policy under BGP peer-group address-family
-*   Validate received capabilities at DUT and ATE reflect support for graceful
-    restart.
-*   For IPv4 and IPv6 routes:
-    *   (Restarting speaker) Advertise prefixes between the ATE ports, through
-        the DUT. Trigger DUT session restart by disconnecting TCP session
-        between DUT and ATE (this may be achieved by using an ACL), determine
-        that packets are:
-        *   Forwarded between ATE port-1 and DUT port-1 for the duration of the
-            specified stale routes time.
-        *   Dropped after the stale routes timer has expired.
-        *   Forwarded again between ATE port-1 and DUT port-1 after the session
-            is re-established.
-    *   (Receiving speaker) Advertise prefixes between the ATE ports, through
-        the DUT. Trigger session restart by disconnecting the BGP session from
-        ATE port-2.
-        *   Ensure that prefixes are propagated to ATE port-2 during the
-            restart.
-        *   Ensure that traffic can be forwarded between ATE port-1 and ATE
-            port-2 during stale routes time.
-        *   Ensure that prefixes are withdrawn, and traffic cannot be forwarded
-            between ATE port-1 and port-2 after the stale routes time expires.
+## Procedure
+ *   Configure EBGP peering between ATE:Port1 and DUT:Port1
+ *   Configure IBGP peering between ATE:Port2 and DUT:Port2
+ *   Ensure that the EBGP and IBGP peering are setup for IPv4-Unicast and IPv6-unicast AFI-SAFIs
```

Comment:
How above are relates to peer-groups? 2 groups, one session each?

Reply:
Correct, one for EBGP and another for IBGP